### DISCUSSION TITLE
Can't Boot Without USB (EFI folder issues)

### DISCUSSION THREAD

**Discussion by Allana-Gibson** [MultimediaLucario/Lenovo-ThinkPad-T440s#33] [Q&A] [answered]
I followed your instructions and successfully launched macOS, but when I shut it down to turn it back on it got stuck on the apple logo. It won’t boot from hard drive only the flash drive. I think im having issues with the EFI folder, because everytime I try to copy it over it says theres not enough space in my EFI partition (it only has 100mb). Is there anything I can do to fix this?

**Comment by MultimediaLucario** [comment, marked as answer]
If you have the OpenCore Configurator app, open it up, select tools in the menu bar, and then mount efi. Next enter your password. Then, go into finder and copy the efi folder from your USB Drive into the drive partition named EFI. Once that is done, restart your computer and remove your USB Drive.

**Reply by Allana-Gibson** [reply]
![Screen Shot 2023-04-17 at 8 42 19 PM](https://user-images.githubusercontent.com/53357849/232640139-bef3684d-2074-4888-aae8-fda74801861d.png)

It wont let me drag the folder into the EFI partition

**Reply by Allana-Gibson** [reply]
![Screen Shot 2023-04-17 at 8 45 48 PM](https://user-images.githubusercontent.com/53357849/232640615-60fe3048-1a2b-4d8e-af20-e8f65816b049.png)

Here are some more screenshots showing that the EFI partiion is only 100MB is this an issue?

**Reply by MultimediaLucario** [reply]
Delete the EFI folder that was in the EFI Partition and empty the trash bin icon. Then drag the EFI folder from the USB on to the EFI Partition of your boot drive.

**Reply by Allana-Gibson** [reply]
It's working now thank you!
